styles/style.css

```diff
@@ -1,10 +1,247 @@
+/*This code is MESSY :( I'll go through and clean it up eventually, but for now, it does the job!*/
+
+
+/*------------------
+| - CSS RESETS -  |
+------------------*/
+* {
+  box-sizing: border-box;
+}
+
+html {
+    font-size: 10px;
+}
+
 body {
-  background: red;
   margin: 0;
-  font-size: 1rem;
-  color: white;
+
+  background: linear-gradient(rgba(0,0,0,0.5),rgba(0,0,0,0.5)),
```

Comment:
You have background set multiple times. Also look at the CSS tricks full size background to get the background constant across resizes of the screen.